scss/grid.scss

```diff
@@ -10,4 +10,4 @@
 
 // Component
 @import "grid/layout";
-@import "grid/theme";
+@import "grid/theme";
```

Comment:
The opposite is specified by the [.editorconfig file](https://github.com/telerik/kendo-theme-default/blob/develop/.editorconfig#L8). Make sure that any used editor [supports it or has a plugin installed](http://editorconfig.org/#download). The reasoning for having these is described in [this SO thread](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

Reply:
As soon as I merge the other request, I will fix this one as well